packages/engine/src/scene/components/MountPointComponent.ts

```diff
@@ -49,18 +50,23 @@ export const MountPointComponent = defineComponent({
   onInit: (entity) => {
     return {
       type: MountPoint.seat as MountPointTypes,
-      helper: null as ArrowHelper | null
+      helper: null as ArrowHelper | null,
+      dismountOffset: new Vector3(0, 0, 0),
+      occupiedAvatarEntity: '' as UserID
```

Comment:
this should be `mountedEntity` with type `EntityUUID` allowing us to mount non-users to mount points